code/mission/missiongoals.cpp

```diff
@@ -1046,7 +1046,9 @@ void mission_eval_goals()
 			}
 
 			// if we get here, then the timestamp on the event has popped -- we should reevaluate
-			PROFILE("Repeating events", mission_process_event(i));
+
+			TRACE_SCOPE(tracing::RepeatingEvents);
+			mission_process_event(i);
```

Comment:
The spacing is the same as it was before my changes. We generally try to keep the style like it was before when changing code.